src/Aspire.Dashboard/Components/Layout/MainLayout.razor

```diff
@@ -31,11 +32,22 @@
             <FluentIcon Value="@(new Icons.Regular.Size24.Settings())" Color="Color.Neutral" />
         </FluentButton>
     </FluentHeader>
-    <NavMenu />
-    <FluentBodyContent Class="custom-body-content">
-        <FluentToastProvider />
-        @Body
-    </FluentBodyContent>
+
+    <AuthorizeView>
```

Comment:
We (at least initially) opted for a full-page login screen, so no header or anything either, so I just switched it to an empty layout template instead and didn't really make use of this. If we keep that other method of doing it, I guess this can be removed?

Reply:
Looks good so far!  I'm struggling with a good title currently but maybe something like ".NET Aspire Dashboard Login"?  It also might be good to have a header in the login prompting the user to input a token because it looks a little barren otherwise.